### ISSUE TITLE
Titaniam is now Portal26 

### ISSUE THREAD

**Issue by pakshi-titaniam** [opensearch-project/project-website#2709]
**NOTE: This is an issue template for new partner pages we are building. Existing partners can use them to submit information needed as we build out enhanced pages. New partners, please continue to use the process outlined here for the time being: https://opensearch.org/new-partner.html

**Organization name**
Portal26, Inc.

**Brief description of your business and what it does (50 words or less)**
Portal26 provides encryption based data protection for big data stores such as AWS S3, OpenSearch, Redshift etc. Portal26 also leverages the above encryption to provide a SaaS platform for customers to monitor and understand data security risks in the GenAI usage within the enterprise.

**Name of your relevant product(s) or service for the OpenSearch audience**
Portal26 Plugin for OpenSearch encrypts and protects data stored in OpenSearch - both at Rest and API results. Customers can supply their own encryption keys at the index level. Available for self-managed open source clusters and also for the managed OpenSearch platform.

**What OpenSearch technologies do you specialize in**
Data Security - Securing the data stored in the indices. If an index is protected by Portal26 no one should be able to bulk exfiltrate the data in clear - neither from the file system or from the API. In addition Portal26 supports BYOK for large Saas operators wherein your customers can host the encryption keys.

**Type of business**
Software Provider

**Home/main office location or address**
15559 Union Avenue, Suite 3054, Los Gatos CA 95032

**How can people contact you (email and phone)**
info@portal26.ai

**Regions you serve**
Worldwide

**Industry specialization**
Portal26's and its products are industry agnostic. That said most requests come from Financial services, Healthcare, Government and SaaS Operators.

**Logo**
![portal26_logo_black_font copy](https://github.com/opensearch-project/project-website/assets/147097978/626ecd1a-4bf9-4cac-9118-f9ef992aff55)

**Additional image**
None

**Links**
(The plugin is referred as _Arcus_)
https://opensearch.org/blog/partner-highlight-titaniam/
https://opensearch.org/blog/partner-highlight-titaniam-byok/

**Comment by pajuric** [comment]
@nateynateynate - Is this something that Jon can complete? The partner's info needs to be updated.

**Comment by kaimmej** [comment]
Instead of 'Software Provider' I have put 'Software Vendor' to be consistent with current Partner business types.
Instead of 'Worldwide' I have put 'Global' for the same reason, consistency with our current data.

Question:
I have maintained the current Titanium product image because a product image was not provided. It uses the previous titanium logo instead of the new Portal26 logo - should this be removed? I renamed it 'portal26-product-image.png' and put it in the new partners asset folder. 

PR created: https://github.com/opensearch-project/project-website/pull/3036
